damus/Util/Utility.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Utility.swift
+//  damus
+//
+//  Created by Swift on 1/14/23.
+//
+
+import SwiftUI
+
+extension String {
+
+    /// Conform plurality of word based on the count
+    /// - Parameter count: Generic type count
+    /// - Returns: String appended with 's' if necessary
+    func conformPlurality<T>(count: T) -> String {
+        var string = self
+        if let value = count as? Int,
+            value != 1 {
```

Comment:
This isn't the right fix. It works for only languages that have the same plurality rules as English.

Here's how I fixed it:
https://github.com/damus-io/damus/blob/cbc3c46c9d554c8829dc7ca9a9ffac48ffbcc3a1/damus/en-US.lproj/Localizable.stringsdict

The docs from Apple describe this in more detail:
https://developer.apple.com/documentation/xcode/localizing-strings-that-contain-plurals